app/main.py

```diff
@@ -2,0 +1,15 @@
+def copy_file(command: str) -> None:
+    split = command.split(" ")
+    command = split[0]
```

Comment:
Ensure that the command is exactly 'cp' and that the input string contains exactly three elements. This is a requirement from the checklist: 'Make sure you check that the input string contains 3 elements separated by space, and has `cp` on a first place.'